docs/_ug/commandSummary/itemCommands/new.md

```diff
@@ -1,2 +1,2 @@
 <!-- markdownlint-disable-file first-line-h1 -->
-<b>new n/ITEM_NAME [qty/QUANTITY] [unit/UNIT] [bgt/BOUGHT_DATE] [exp/EXPIRY_DATE]</b><br><br><u>Valid Example:</u><br>new n/Potato qty/70 unit/kg bgt/22-02-11 exp/22-03/11
+`new n/ITEM_NAME [qty/QUANTITY] [unit/UNIT] [bgt/BOUGHT_DATE] [exp/EXPIRY_DATE]`
```

Comment:
Ditto on consistency. `ITEM_NAME` or `n/ITEM_NAME`? Compare with `find` command.

Reply:
Ah actually, certain commands require the delimiter n/, some commands that only have one input (e.g. the Find command) will only need one input, and does not have the delimiter